parsel/selector.py

```diff
@@ -139,9 +139,9 @@ class Selector(object):
     selectorlist_cls = SelectorList
 
     def __init__(self, text=None, type=None, namespaces=None, root=None,
-                 base_url=None, _expr=None):
+                 base_url=None, _expr=None, parser_cls=None):
```

Comment:
hm, I still don't like that create_root_node assumes a very specific HTMLParser class; it doesn't work with stdlib HTMLParser, it doesn't work with html5lib HTMLParser, and there is no parser class in lxml.html.soupparser.